scripts/format-code

```diff
@@ -190,10 +189,10 @@ get_find_args()
     local defaultIncludeExts='h c cpp'
 
     if [[ -z ${userExcludeDirs} ]]; then
-        local excludeDirs=( ${defaultExcludeDirs} )
+        local excludeDirs=( "${defaultExcludeDirs}" )
```

Comment:
I’m no expert, but aren’t these breaking changes? Instead of an array of directories, the introduced quotes will make it it an array of one element, a string of space separated directory names, I think.

Reply:
ah, you're right. I believe this was reported by the `check-linters` script. I'll revert back and take another look at why.